docs/examples/000/1. hello-world/readme.md

```diff
@@ -0,0 +1,96 @@
+# Hello-world sample
+This bicep sample is the canonical hello world.
+
+## Code description
+This bicep file takes a `yourName` parameter and adds that to a `hello` variable and
+returns the concatenated string as an ARM output.
+
+## Deploy process
+The command to deploy this bicep file is:
+
+### PowerShell

```

Comment:
These deployment commands are duplicated in the comment above main.bicep and here. Personally, I prefer it in here in the readme so the `.bicep` file is cleaner. Thoughts?

Reply:
Sure - I have no problems with them only in the readme.md file.